admin-dev/themes/new-theme/template/components/layout/nav_bar.tpl

```diff
@@ -22,7 +22,7 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  *}
-<nav class="nav-bar d-none d-md-block">
+<nav class="nav-bar d-none d-print-none d-md-block">
```

Comment:
Shouldn't this be handled using css selectors?

Reply:
It should be, but if we decide that then everyone in the team should now adopt a complete component based guideline which permit every single component to be selected easily, even .row, .container, .col-md-* and stop using m*-* like classes but prefer creating a stylesheet for every components right named, else that would be a nonsense, some PR's would have this review, and others not

In that case that could be done CSS's side easily, I agree !